jetty-http-spi/pom.xml

```diff
@@ -35,6 +35,16 @@
       <version>${project.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <scope>provided</scope>
```

Comment:
why provided here?

Reply:
Ditto

Reply:
Historically, there's been no hard dependencies in this project.
See for yourself, there's no compile/runtime dependencies.

Reply:
@joakime if there is a test failure and I want to debug it, why should I change the POM? I would just have `scope=test` like everywhere else.

Reply:
If there is a test failure and you want to debug it, don't change the pom.

On maven slf4j-api + jetty-slf4j-impl + jul-to-slf4j are all included in this project.
In your IDE slf4j-api + jetty-slf4j-impl + jul-to-slf4j  are all included in this project.

It's just not included when you USE the `jetty-http-spi`, and like all of the other dependencies in `jetty-http-spi`, nothing is included, intentionally.